### PR TITLE
Add tenant parameter to shared url

### PR DESCRIPTION
**Pull request by zengyan-amazon**
*Issue #, if available:* #591

*Description of changes:*
* add `security_tenant=<tenant_name>` as query parameter to shared urls e.g. `http://localhost:5601/xdj/goto/81eb5d0c993fc4d4a37713383368abb5?security_tenant=global`
* refresh page when admin switch tenant on tenant list page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/services/shared-link.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+import { parse } from 'url';
+import { CoreStart } from 'kibana/public';
+import { API_ENDPOINT_MULTITENANCY } from '../apps/configuration/constants';
+
+// refer to https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/master/public/chrome/multitenancy/enable_multitenancy.js
+export async function addTenantToShareURL(core: CoreStart) {
+  let tenant = '';
+  try {
+    tenant = await core.http.get(API_ENDPOINT_MULTITENANCY);
```
Comment (yuxitang-amzn) [inline public/services/shared-link.ts:24]: Will this run before login? Do we need to ignore 403?

File: public/apps/configuration/panels/tenant-list/tenant-list.tsx [hunk 2]
```diff
@@ -131,12 +131,7 @@ export function TenantList(props: AppDependencies) {
   const switchToSelectedTenant = async (tenantValue: string, tenantName: string) => {
     try {
       await changeTenant(tenantValue);
-      setSelection([]);
-      addToast({
-        id: 'selectSucceeded',
-        title: `Selected tenant is now ${tenantName}`,
-        color: 'success',
-      });
+      window.location.reload();
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/tenant-list/tenant-list.tsx:134]: Please add some detail why we want to refresh here.

**Review by yuxitang-amzn — APPROVED** [review]
Overall LGTM, some minor comments.

File: public/services/shared-link.ts (hunk 1 above)
Comment (zengyan-amazon) [reply public/services/shared-link.ts:24]: yes it execute on login page, we log the error in console and do nothing here, like what we did for permissionInfo call

**Pull request merged**